docs/calicoctl/resources/policy.md

```diff
@@ -0,0 +1,93 @@
+# Policy Resource
+Policy objects can be thought of as being applied to a set of endpoints (rather than being a property of the endpoint) to give more flexible policy arrangements, including support for multiple tiers of policy (e.g. net sec, ops, dev team) that can override or augment any ACLs directly associated with an endpoint through a profile.
+
+Each policy has a label/tag based selector predicate, such as “type == ‘webserver’ && role == ‘frontend’”, that selects which endpoints it should apply to, and an ordering number that specifies the policy’s priority. For each endpoint, Calico applies the security policies that apply to it, in priority order, and then that endpoint’s security profiles.
+
+### Sample YAML
+```
+apiVersion: v1
+kind: policy
+metadata:
+  name: policy1
+  tier: tier1
+spec:
+  order: 100
+  selector: type=='database'
+  ingress:
+  - action: deny
+    protocol: tcp
+    icmp:
+      - type: 10
+      - code: 6
+    "!protocol": ipv6
+    "!icmp":
+      - type: 19
+      - code: 255
+    source:
+      tag: footag
```

Comment:
@robbrockbank Am I right here? [Tags are just strings](https://github.com/tigera/libcalico-go/blob/master/lib/api/rule.go#L46). Is this a legitimate sample?